lib/testing/src/lib/core/pages/card-view/card-view-text-item.page.ts

```diff
@@ -70,4 +70,8 @@ export class CardTextItemPage {
     async checkElementIsReadonly(): Promise<void> {
         await BrowserVisibility.waitUntilElementIsVisible(this.rootElement.element(this.readOnlyField));
     }
+
+    async clickElement(): Promise<void> {
```

Comment:
which element are you clicking?

Reply:
If you refer to the name of the method, 'element' refers to 'card-text-item'

Reply:
better to specify what is clicking there are multiple thing that can be click I guess